payloads/start.go

```diff
@@ -86,17 +86,35 @@ const (
 	Docker = "docker"
 )
 
-// StorageResources represents a requested storage resource for a workload.
-type StorageResources struct {
+// StorageResource represents a requested storage resource for a workload.
```

Comment:
I agree with @rbradford that we should name this something else to avoid confusion in controller between payloads.StorageResource and types.StorageResource.